bin/build_daq_container

```diff
@@ -15,3 +15,5 @@ echo Starting $image_name build...
 docker build . -f docker/modules/Dockerfile.daq --build-arg GIT_URL -t daqf/$image_name
 
 echo Done with $image_name build.
+
+docker build . -f subset/pentests/nmap/Dockerfile.test_nmap -t daqf/nmap
```

Comment:
missing newline

Reply:
This was an accidental file change - modification removed